### DISCUSSION TITLE
How to download the .sppkg release package

### DISCUSSION THREAD

**Discussion by romcode** [microsoft-search/pnp-modern-search#1930] [Q&A] [answered]
Hi,
anyone can show me how to download the the release package ?

Documentation states in https://microsoft-search.github.io/pnp-modern-search/installation/ that it should be done from the release page, but I can't find the download link.

Any help would be appreciated.

Best regards.

**Comment by wobba** [comment]
Go to https://github.com/microsoft-search/pnp-modern-search/releases. The link is available on the start page at https://github.com/microsoft-search/pnp-modern-search in the right hand column.

**Reply by romcode** [reply]
Sorry @wobba, but I can't see any download link in the release page. I attach screenshot.

![image](https://user-images.githubusercontent.com/33631197/158074104-43e2abc8-4409-42ed-a7e2-c143d3b6b8c4.png)

**Comment by wobba** [comment, marked as answer]
Scroll a tiny bit more and you see a section called "Assets" per release
with the sppkg to download.

**Reply by romcode** [reply]
Great! It's there! 
Thank you so much for your help.
Best regards.
